flink-runtime/src/test/java/org/apache/flink/runtime/jobmanager/scheduler/SchedulerIsolatedTasksTest.java

```diff
@@ -294,11 +297,11 @@ public void testScheduleWithDyingInstances() {
 			
 			i2.markDead();
 			
-			for (SimpleSlot slot : slots) {
-				if (slot.getOwner() == i2) {
-					assertTrue(slot.isCanceled());
+			for (LogicalSlot slot : slots) {
+				if (Objects.equals(slot.getTaskManagerLocation().getResourceID(), i2.getTaskManagerID())) {
```

Comment:
I think `slot.getTaskManagerLocation().getResourceID()` cannot return null. Is there a need to use `Objects.equals`?

Reply:
No specific reason. Will change it.